.bazelrc

```diff
@@ -53,17 +53,15 @@ build --cxxopt -Wformat-security
 # Link pthread for flatbuffers
 build --host_linkopt=-pthread
 
+build:clang --host_action_env=CC=
+build:clang --host_action_env=CXX=
```

Comment:
not sure why need this, cc @kyessenov

Reply:
I see, so this is an environment pollution I think from Istio CI.